build.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
+
```

Comment:
@mgsdk I put this back, whilst we can explicitly throw errors it's nice also not to silently fail